template/.github/pr-verify.yml

```diff
@@ -0,0 +1,23 @@
+name: PR Verify
```

Comment:
I like to have a check like this in all my fresh projects but I can remove this if we don't want it in the template